src/lib/dialog_ui.sh

```diff
@@ -99,6 +99,8 @@ function create_menu_options()
   cmd+=" '${height}' '${width}' '${max_elements_displayed_in_the_menu}'"
 
   for item in "${_menu_list_string_array[@]}"; do
+    # Work-around to prevent bugs
+    item=$(printf '%s' "$item" | sed -e "s/\"/'/g" -e "s/\`/'/g")
```

Comment:
With my previous comments, we could simplify this part by using something like `sed -e "s/'/\\\'/g"`. I guess this address the single-quotes problem. Ideally, we don't want this change here; we want to move it to the lore file. Or we can create another dedicated function to process the item just for the dialog code.